README.md

```diff
@@ -0,0 +1,357 @@
+
+There are two ways of managing the ACL DB contents:
+- Through the use of [ERL commands](#aclstore_commands) from the RabbitMQ Erlang console.
+- Through the use of the [AMQP Administration API](#amqp_administration).
+
+### <a name="examples"/> Examples of use
+
+Once the plugin is deployed and configured, and the ACL established, the plugin will start enforcing the authorizations.
+The `/docs` folder contains a series of examples of use that can be used to illustrate the behavior of the RabbitMQ broker
+with the activated plugin.
+
+Some examples make use of the [RabbitMQ ACL Tool](https://github.com/dmoranj/rabbit-acl-tool) commands for testing
+purposes. Check the documentation in the repository for further information.
+
+#### Example 1
+
+This example shows the basic behavior of the plugins for a very simple scenario: a news feed for a newspaper. The news
+feed has two kinds of topics:
+
+* `news/categories/<category>`: news channels for the different news categories offered by the newspaper. Journalists publish and
+read from this resource and the general audience subscribes to the messages.
+* `news/events`: private channel for the newspaper where the editor publishes events that must be covered by the journalists.
+ The editor publish to this resource and the journalists read from it.
+
+All the messages will be published to the `newsfeed` exchange.
+
+All the files for this example can be found in the `/docs/example1` folder.
+
+In order to setup the initial scenario, the RabbitMQ users must be created in the broker. Go to the `bin/` folder of
+your RabbitMQ instance and execute the `setupTest.sh` script. This script will create three users, on for each role in
+our example app: journalist, audience and editor. It also gives permissions for accessing all kind of entities in the
+default VHost. Take into account that this permissions are used at a different level than the routing key permissions
+offered by the plugin: this plugin does not override default RabbitMQ security, it adds a new level of security that
+takes into account not only RabbitMQ resources, but also routing keys.
+
+Once the users have been created in the broker, it's time to load the ACL file. There is an example ACL file called
+`newsacl` in the `/docs/example1` folder, with the permissions described in the problem description.
+
+The file can be loaded using the RabbitMQ Erlang console. If you are testing the plugin using the `make run-broker`
+build task, show the Erlang console by pressing Enter. If you are using a standalone RabbitMQ, you can issue Erlang
+commands to the server through the `rabbitmqctl eval <expression>` command. In the following, we will use the latter
+method (in order to use the former, just issue the expressions directly through the Erlang console).
+
+Use the following command to show the current ACL DB contents:
+
+```
+./rabbitmqctl eval "aclstore:list_permissions()."
```

Comment:
Depending on how did you install RabbitMQ. If you installed it system-wide (e.g.: with APT in an Ubuntu system), there will be no './' prefix in the command. For local installations (as the one used in the tests), you would have to execute it from the `bin/` folder of the local instance, so it would require the prefix. NTC